checklist.md

```diff
@@ -0,0 +1,118 @@
+# Сheck Your Code Against the Following Points
+
+## Don't Push db files
+
+Make sure you don't push db files (files with `.sqlite`, `.db3`, etc. extension).
+Don't forget to add `.gitignore` file before pushing. 
```

Comment:
May be swich:
## Don't forget to add `.gitignore` file before pushing. 
Make sure you don't push db files (files with `.sqlite`, `.db3`, etc. extension).

Reply:
and even BEFORE